coderd/database/migrations/000301_add_workspace_app_audit_sessions.up.sql

```diff
@@ -0,0 +1,27 @@
+CREATE UNLOGGED TABLE workspace_app_audit_sessions (
```

Comment:
Review: Note `UNLOGGED`, we don't consider these session critical.

We may want to remove/clean them up in the future as well.

Reply:
TIL!